incubator/kafka/templates/statefulset.yaml

```diff
@@ -25,7 +25,7 @@ spec:
             command:
               - bin/kafka-topics.sh
               - --zookeeper
-              - "{{ template "name" . }}-zookeeper:2181"
+              - "{{ .Release.Name }}-zookeeper:2181"
```

Comment:
* `fullname` would generate something like `eloping-anaconda-kafka-zookeeper:2181` where `eloping-anaconda` is a release name.

* Yes, it works without `--name` since the `.Release.Name` is populated by helm before templating the files (as I understand).